website/cue/reference/cli.cue

```diff
@@ -589,6 +596,13 @@ cli: {
 			description: "Watch for changes in the configuration file and reload accordingly"
 			type: bool: default: false
 		}
+		VECTOR_INTERNAL_LOG_RATE_LIMIT: {
+			description: "Set the internal log rate limit"
```

Comment:
It would probably be worth expanding this to help users understand why they would want a rate limit. ie something like "This limits Vector from emitting identical logs more than once over the given number of seconds."